### PR TITLE
Replaced calls to panic with returned error

### PR DESCRIPTION
**Pull request by MarkSonghurst**
As requested in #77

### CODE REVIEW

File: handle.go [hunk 1]
```diff
@@ -10,7 +10,8 @@ import (
 	"github.com/alexbrainman/odbc/api"
 )
 
-func ToHandleAndType(handle interface{}) (h api.SQLHANDLE, ht api.SQLSMALLINT) {
+func ToHandleAndType(handle interface{}) (h api.SQLHANDLE, ht api.SQLSMALLINT, err error) {
+	err = nil
```
Comment (alexbrainman) [inline handle.go]: Remove line 14. err is initialized to nil already.

**Comment by MarkSonghurst** [comment]
Thanks for the review, I have made the requested change.

**Comment by alexbrainman** [comment]
LGTM

Thank you.

Alex

**Comment by MarkSonghurst** [comment]
@alexbrainman I think something strange has happened - when I pull master, I cannot see my changes and the panic calls remain?
Also browsing in github shows the same - for example on line 61:
https://github.com/alexbrainman/odbc/blob/master/error.go

**Comment by alexbrainman** [comment]
> I think something strange has happened - when I pull master, I cannot see my changes and the panic calls remain?
> Also browsing in github shows the same - for example on line 61:
> https://github.com/alexbrainman/odbc/blob/master/error.go

I can see your change here https://github.com/alexbrainman/odbc/commit/bfee230746b69a0370e3726f11e92faea027ca11

And when I pull from master I can see your changes on my computer all right.

Alex

**Comment by MarkSonghurst** [comment]
@alexbrainman Thanks - but it's still wrong for me:
```
[msonghurst@deimos test]$ git clone git@github.com:alexbrainman/odbc.git
Cloning into 'odbc'...
remote: Counting objects: 365, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 365 (delta 2), reused 4 (delta 1), pack-reused 352
Receiving objects: 100% (365/365), 136.69 KiB | 331.00 KiB/s, done.
Resolving deltas: 100% (230/230), done.
[msonghurst@deimos test]$ grep -R panic *
odbc/column.go:		panic(fmt.Errorf("do not know how wide column of ctype %d is", ctype))
odbc/column.go:		panic(fmt.Errorf("wrong column #%d length %d returned, %d expected", idx, c.Len, c.Size))
odbc/driver.go:		panic(err)
odbc/error.go:			panic(fmt.Errorf("SQLGetDiagRec failed: ret=%d", ret))
odbc/mssql_test.go:				panic(err)
```

Any suggestions please?
I have deleted my fork as I now want to use yours, but I can't see that causing this issue.

**Comment by MarkSonghurst** [comment]
Also if you click on https://github.com/alexbrainman/odbc/blob/master/error.go can you see line 61 with the panic still?

**Comment by alexbrainman** [comment]
> Also if you click on https://github.com/alexbrainman/odbc/blob/master/error.go can you see line 61 with the panic still?

But your PR did not change line 61 in error.go. See for yourself there https://github.com/alexbrainman/odbc/pull/104/files

Alex

**Comment by MarkSonghurst** [comment]
Ah, you are right! My bad, sorry. Not a problem.
Thanks for your time.

**Comment by MarkSonghurst** [comment]
@alexbrainman I think some of my panic/error changes got lost when I squashed commits.
I'll raise another PR with the missing changes in. 
I'm very sorry for the inconvenience.

**Comment by alexbrainman** [comment]
> I'll raise another PR with the missing changes in.

Sounds good.

> I'm very sorry for the inconvenience.

No worries.

Alex

**Pull request merged**